src/Features/CSharp/Portable/CodeRefactorings/InlineTemporary/InlineTemporaryCodeRefactoringProvider.InitializerRewriter.cs

```diff
@@ -1,103 +0,0 @@
-﻿// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
remove entirely.  this code existed just for the case of handling a varaible initialized with a reference to itself.